collectors/deployment.go

```diff
@@ -177,9 +186,10 @@ func (dc *deploymentCollector) collectDeployment(ch chan<- prometheus.Metric, d
 		addGauge(descDeploymentCreated, float64(d.CreationTimestamp.Unix()))
 	}
 	addGauge(descDeploymentStatusReplicas, float64(d.Status.Replicas))
+	addGauge(descDeploymentStatusReplicasUpdated, float64(d.Status.UpdatedReplicas))
+	addGauge(descDeploymentStatusReplicasReady, float64(d.Status.ReadyReplicas))
```

Comment:
Why should we move `descDeploymentStatusReplicasUpdated` here?

Reply:
I corrected `code` location in order to be consistent with the `deployment REST API response`